app/models/opensearch.rb

```diff
@@ -109,9 +109,30 @@ def matches
     match_single_field_nested(:identifiers, m)
     match_single_field_nested(:locations, m)
     match_single_field_nested(:subjects, m)
+
+    match_geopoint(m) if @params[:geopoint].present?
```

Comment:
@matt-bernhardt did we agree on `geodistance` for this parameter? If so I can update this. I believe my code is from before we met to discuss naming for this feature.

Reply:
I think I want to make something work before we bring them in on naming. I think since OpenSearch calls this a geodistance search we had agreed to that and I just hadn't updated the code. I'll make this `geodistance` as the parameter name we agree to for now and once implemented fully in graphql we can get feedback. Thanks!